Source/Scene/Scene.js

```diff
@@ -1659,6 +1716,108 @@ define([
         }
     }
 
+    function insertShadowCommands(scene, commandList, insertAll, shadowMap, shadowPassCommands) {
```

Comment:
Likewise, when commands come from a spatial data structure - like terrain and 3D Tiles - could these checks be any more efficient?